accounts/keystore/keystore.go

```diff
@@ -67,7 +67,8 @@ type KeyStore struct {
 	updateScope event.SubscriptionScope // Subscription scope tracking current live listeners
 	updating    bool                    // Whether the event notification loop is running
 
-	mu sync.RWMutex
+	mu    sync.RWMutex
+	impMu sync.RWMutex // Import Mutex locks the import to prevent two insertions from racing
```

Comment:
I'd prefer it written out, spend another two characters for better readability: `importMu`